nntrainer/layers/nnstreamer_layer.cpp

```diff
@@ -81,8 +81,11 @@ static int nnst_info_to_tensor_dim(ml_tensors_info_h &out_res, TensorDim &dim) {
   if (type != ML_TENSOR_TYPE_FLOAT32)
     return ML_ERROR_NOT_SUPPORTED;
 
-  if (ML_TENSOR_RANK_LIMIT > ml::train::TensorDim::MAXDIM)
-    return ML_ERROR_NOT_SUPPORTED;
+  // @todo : This is just temp fixes. After nnstreamer find the way to get the
+  // rank
+
+  // if (ML_TENSOR_RANK_LIMIT > ml::train::TensorDim::MAXDIM)
+  //   return ML_ERROR_NOT_SUPPORTED;
```

Comment:
We can check it (after LINE90):
```
if (ML_TENSOR_RANK_LIMIT > ml::train::TensorDim::MAXDIM) {
    unsigned int i;
    for (i = ml::train::TensorDim::MAXDIM; i < ML_TENSOR_RANK_LIMIT; i++)
        if (dim_[ml::train::TensorDim::MAXDIM] > 1)
            return ML_ERROR_NOT_SUPPORTED; /* A more detailed description of the error would be nice though.. */
}

```

Reply:
If dim_[ML_TENOSR_RANK_LIMIT] is, it will work. Thanks.

Reply:
On second thought, it cannot distinguish rank 5 with ```2:3:4:5:1```.